support-frontend/assets/pages/contributions-landing/components/ContributionAmount.jsx

```diff
@@ -126,21 +126,17 @@ function withProps(props: PropTypes) {
     <ContributionTextInputDs
       id="contributionOther"
       name="contribution-other-amount"
-      type="number"
```

Comment:
Defining this field as type="number" gets in the way of our own validation. It prevents the onUpdate callback from being called if e.g. the user enters a char